draft-ietf-quic-qpack.md

```diff
@@ -215,12 +218,15 @@ been referenced can still be blocking.
 Note:
 : A blocking entry is unrelated to a blocked stream, which is a stream that a
   decoder cannot decode as a result of references to entries that are not yet
-  available.  Any encoder that uses the dynamic table has to keep track of
-  blocked entries, whereas blocked streams are optional.
+  available.  An encoder that uses the dynamic table has to keep track of
+  blocked entries.
 
 An encoder MUST NOT insert an entry into the dynamic table (or duplicate an
-existing entry) if doing so would evict a blocking entry.  In this case, the
-encoder can send literal representations of header fields.
+existing entry) if doing so would evict a blocking entry.  An encoder can always
+send literal representations of header fields when the dynamic table is full
```

Comment:
Perhaps just omit "when the dynamic table..."?  It's true universally, regardless of the dynamic table state.

Reply:
Maybe, but giving them a hint seems reasonable and generous.

Reply:
This is covered in the "Encoder" section: 

> An encoder converts a header list into a header block by emitting either an indexed or a literal representation for each header field in the list
...
References to the static table and literal representations do not require any dynamic state and never risk head-of-line blocking

I guess it doesn't specifically state the encoder "can always send" these representations, but I'm going for less=more here.  Maybe we can add it later, if really needed.  I don't think it belongs in Avoiding Blocked Insertions, since optimistic insertions could also be blocked, and have nothing to do with literal representations.